### DISCUSSION TITLE
translation warning during rendering

### DISCUSSION THREAD

**Discussion by sbalci** [quarto-dev/quarto-cli#6039] [Q&A] [answered]
### Description

I am rendering a book in Turkish and I get the following error. Is there something missing in `translations/tr.yaml`. I can make the necessary translation if you guide me with the error.

Here is the `yml` location for translations:
https://github.com/patolojiatlasi/patolojiatlasi.github.io/blob/main/_quarto.yml#L22
https://github.com/patolojiatlasi/patolojiatlasi.github.io/blob/main/_quarto.yml#L24

```
[WARNING] Could not load translations for tr
  translations/tr.yaml: Aeson exception:
  Error in $: Invalid Term name "Subject"
[WARNING] The term Abstract has no translation defined.
```

**Comment by cderv** [comment]
First our language doc is here: https://quarto.org/docs/authoring/language.html

Setting the `lang: tr` should activate the language file (https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/language/_language-tr.yml) 

`Abstract` should be translated per 
https://github.com/quarto-dev/quarto-cli/blob/4b12deeb01eb9a40e2a4b2f87d24118455f61ef1/src/resources/language/_language-tr.yml#L14

However, I think the error comes from Pandoc somehow as the `tr.yaml` file is this one: https://github.com/jgm/pandoc/blob/main/data/translations/tr.yaml

I would open an issue in Pandoc itself, because it seems the `tr.yaml` file as one more line than the other (the additional being `Subject`) 
https://github.com/jgm/pandoc/blob/f074b03b899cf2ca5e9574e17fcc9ab2a40019ec/data/translations/tr.yaml#L20

I think you would get the issue with Pandoc itself without Quarto. Take this simple document

````markdown
---
lang: tr
title: Dizin
---

# Dizin
````

```powershell
> pandoc -t html -f markdown test2.qmd
[WARNING] Could not load translations for tr
  translations/tr.yaml: Aeson exception:
  Error in $: Invalid Term name "Subject"
[WARNING] The term Abstract has no translation defined.
<h1 id="dizin">Dizin</h1>
```

so it seems this is an issue with `lang: tr` and Pandoc. 

You should open an issue there

**Comment by dragonstyle** [comment, marked as answer]
This looks like an issue in Pandoc. I've reported it there and you can track it with this issue:

https://github.com/jgm/pandoc/issues/8921

Thanks for letting us know!

**Reply by cderv** [reply]
Thanks @dragonstyle !
